### PR TITLE
Settings and some fix from the last pull request

### PR DESCRIPTION
**Pull request by Alex-git-code**
Settings and some fix from the last pull request

### CODE REVIEW

File: app/src/main/java/org/alexcode/snake/EditNameDialog.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public void onClick(DialogInterface dialogInterface, int i) {
                             Toast.makeText(getContext(), "Player name maximum length can be 15 characters. including spaces.", Toast.LENGTH_SHORT).show();
                         } else {
                             playerId = PlayerPreferences.getPlayerId();
-                            if (!newPlayerName.equals(playerName)) {
```
Comment (andreisamoila74) [inline app/src/main/java/org/alexcode/snake/EditNameDialog.java:59]: this shouldn't be modified

File: app/src/main/java/org/alexcode/snake/PlayerPreferences.java [hunk 2]
```diff
@@ -54,14 +56,36 @@ public static int getHiScore() {
         return sharedPreferences.getInt("HI_SCORE", 0);
     }
 
-    public static String getVolume() {
```
Comment (andreisamoila74) [inline app/src/main/java/org/alexcode/snake/PlayerPreferences.java:57]: why did you remove the volume though?

File: app/src/main/java/org/alexcode/snake/Settings.java [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.CompoundButton;
+import android.widget.Spinner;
+import android.widget.Switch;
```
Comment (andreisamoila74) [inline app/src/main/java/org/alexcode/snake/Settings.java:13]: maybe import android.widget.*?

File: app/src/main/java/org/alexcode/snake/Settings.java [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+            case "Spanish":
+                languageSpinner.setSelection(1);
+                break;
+            case "German":
+                languageSpinner.setSelection(2);
+                break;
+            default:
+                languageSpinner.setSelection(3);
+                break;
+        }
```
Comment (andreisamoila74) [inline app/src/main/java/org/alexcode/snake/Settings.java:70]: isn't there a package that ensures all need for the language part? because the language part has to be something very usual in mobile development

**Review by andreisamoila74 — CHANGES_REQUESTED** [review]
these fixes shouldn't be here

File: app/src/main/java/org/alexcode/snake/PlayerPreferences.java (hunk 2 above)
Comment (Alex-git-code) [reply app/src/main/java/org/alexcode/snake/PlayerPreferences.java:57]: I split the sounds part in two: game music and fx sounds.

File: app/src/main/java/org/alexcode/snake/Settings.java (hunk 3 above)
Comment (Alex-git-code) [reply app/src/main/java/org/alexcode/snake/Settings.java:13]: Ok, but I don't know if is a good idea to import all subfolders for some from the point of view of resource usage.

File: app/src/main/java/org/alexcode/snake/Settings.java (hunk 4 above)
Comment (Alex-git-code) [reply app/src/main/java/org/alexcode/snake/Settings.java:70]: You can get the language of the device but you need to make the translations. https://developer.android.com/training/basics/supporting-devices/languages 
I choose to let the user set the language for the application no matter what language settings have the device.
